packages/grafana-alerting/package.json

```diff
@@ -40,24 +40,30 @@
     "codegen": "rtk-query-codegen-openapi ./scripts/codegen.ts"
   },
   "devDependencies": {
+    "@faker-js/faker": "^9.7.0",
+    "@grafana/test-utils": "workspace:*",
     "@grafana/tsconfig": "^2.0.0",
     "@rtk-query/codegen-openapi": "^2.0.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.3.0",
+    "@testing-library/user-event": "^14.6.1",
     "@types/lodash": "^4",
     "@types/react": "18.3.18",
     "@types/react-dom": "18.3.5",
     "react": "18.3.1",
     "react-dom": "18.3.1",
+    "react-redux": "^9.2.0",
     "type-fest": "^4.40.0",
     "typescript": "5.7.3"
   },
   "peerDependencies": {
     "@grafana/runtime": "^12.0.0-pre",
     "@grafana/ui": "^12.0.0-pre",
+    "@reduxjs/toolkit": "^2.8.0",
```

Comment:
This made more sense to me but feel free to let me know if this wouldn't work for consumers

Reply:
Yep, it's both a peer dependency and a dev dependency (ie. won't be in the production bundle, the dev dependency is used for test runners and for prod it will use peer dependencies).

This is also how it's defined in `@grafana/ui` :)